packages/thrift-client/src/main/connections/Connection.ts

```diff
@@ -177,10 +177,14 @@ export class Connection {
                         }
                     }
                 } catch (err) {
+                    let message = ''
```

Comment:
I'm wondering if there's a better pattern we should use here. Like use "Unknown" if it's not an instanceof Error? Or `err = message`?

Reply:
Sure, I'll make the default message be `Unknown`, seems like a decent idea.